### PR TITLE
Feature/fix map reducer state reference

### PR DESCRIPTION
**Pull request by vedrani**
Fix on fix

### CODE REVIEW

File: src/mapReducers.js [hunk 1]
```diff
@@ -40,7 +41,7 @@ function calculateNewStateForAllKeys(state, action, resolveReducer) {
     return reducer(stateForKey, action);
```
Comment (zrumenjak) [inline src/mapReducers.js:41]: Could you just call `calculateNewStateForKey` here for each key?

Comment (zrumenjak) [reply src/mapReducers.js:41]: Also, you are still using `_.eq` in `calculateNewStateForKey `...

Comment (vedrani) [reply src/mapReducers.js:41]: I don't use `eq`, I use `import shallowEqual from 'shallowequal';`

Ok, I will reuse that method. Also, I will add the possibility that key extractor returns an array of keys.

**Review by damirjuretic — APPROVED** [review]
Looks good

**Comment by zrumenjak** [comment]
😂

**Comment by vedrani** [comment]
Cool

**Pull request merged**